Mathlib/Analysis/Complex/TaylorSeries.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2024 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import Mathlib.Analysis.Complex.CauchyIntegral
+
+/-!
+# Convergence of Taylor series of holomorphic functions
+
+We show that the Taylor series around some point `c : ℂ` of a function `f` that is complex
+differentiable on the open ball of radius `r` around `c` converges to `f` on that open ball;
+see `Complex.hasSum_taylorSeries_on_ball` and `Complex.taylorSeries_eq_on_ball` for versions
+(in terms of `HasSum` and `tsum`, repsectively) for functions to a complete normed
+space over `ℂ`, and `Complex.taylorSeries_eq_on_ball'` for a variant when `f : ℂ → ℂ`.
+
+We also show that the Taylor series around some point `c : ℂ` of a function `f` that is complex
+differentiable on all of `ℂ` converges to  `f` on `ℂ`;
+see `Complex.hasSum_taylorSeries_of_entire`, `Complex.taylorSeries_eq_of_entire` and
+`Complex.taylorSeries_eq_of_entire'`.
+-/
+
+namespace Complex
+
+open BigOperators Nat
+
+variable {E : Type*} [NormedAddCommGroup E] [NormedSpace ℂ E] [CompleteSpace E] ⦃f : ℂ → E⦄
+
+section ball
+
+variable ⦃c : ℂ⦄ ⦃r : NNReal⦄ (hf : DifferentiableOn ℂ f (Metric.ball c r))
```

Comment:
Can you add versions with `EMetric.ball`? And why not allow `r` to be a real? EMetric.ball takes an ENNReal but Metric.ball takes a Real.

Reply:
Done.